src/py21cmfast/inputs.py

```diff
@@ -452,6 +452,9 @@ class UserParams(StructWithDefaults):
         is considerably faster than when performing integrals explicitly.
     FAST_FCOLL_TABLES: bool, optional
         Whether to use fast Fcoll tables, as described in Sec X of JBM XX. Significant speedup for minihaloes.
+    MINIMIZE_MEMORY: bool, optional
+        If set, the code will run in a mode that minimizes memory usage, at the expense
+        of some CPU/disk-IO. Good for large boxes / small computers.
```

Comment:
no expense of disk-IO since you are not reading in unfiltered_box from the disk again.

Reply:
But this is the same flag as created by @steven-murray with #220. Therefore, there will be additional IO.

Reply:
Yup, this is used on the python side as well, and will involve extra IO.